contracts/modules/royalty/RoyaltyModule.sol

```diff
@@ -66,14 +69,22 @@ contract RoyaltyModule is
         _;
     }
 
-    /// @notice Sets the license registry
+    /// @notice Sets the licensing module
     /// @dev Enforced to be only callable by the protocol admin
-    /// @param licensing The address of the license registry
+    /// @param licensing The address of the license module
     function setLicensingModule(address licensing) external onlyProtocolAdmin {
         if (licensing == address(0)) revert Errors.RoyaltyModule__ZeroLicensingModule();
         _getRoyaltyModuleStorage().licensingModule = licensing;
     }
 
+    /// @notice Sets the dispute module
+    /// @dev Enforced to be only callable by the protocol admin
+    /// @param dispute The address of the dispute module
+    function setDisputeModule(address dispute) external onlyProtocolAdmin {
+        if (dispute == address(0)) revert Errors.RoyaltyModule__ZeroDisputeModule();
+        _getRoyaltyModuleStorage().disputeModule = dispute;
+    }
+
```

Comment:
Instead of making this "set" function as a governance function. 
It would better follow the "update" pattern of updating the dispute module address according to ModuleRegistry.
We have apply the pattern in the CoreMedataViewModule:
```solidity
    function updateCoreMetadataModule() external {
        coreMetadataModule = IModuleRegistry(MODULE_REGISTRY).getModule(CORE_METADATA_MODULE_KEY);
    }
```
in this case, it would be:
```solidity
    function updateDisputeModule() external {
         _getRoyaltyModuleStorage().disputeModule = IModuleRegistry(MODULE_REGISTRY).getModule(DISPUTE_MODULE_KEY);
    }
```    
so that the function can be **permissionless** and maintain code style consistency in our code base.

Reply:
I would not advertise this as a "permissionless" method, because the setter of value in ModuleRegistry is still the governance. Simply delegating the reading & setting to the public, where the value is controlled by the governance. 

But yes, this is a cleaner syntax since we don't need to pass in the dispute module address for each contract. Just update in the Module Registry, and then call `updateDisputeModule` in each relevant contract.

I recommend we make this change for all contracts in a separate PR, since most other modules are still using the old style.

Reply:
agree,  we will leave those changes to next PR @Spablob